miscellaneous/build_templates/analysis.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     steps:
       # Remove ".placeholder" from filenames
       - bash: |
-          for filename in *
+          for filename in $(find -name '*')
```

Comment:
Is it possible to do something more like `find -name '*.placeholder*'`?

Reply:
That's a bit more elegant so I made the change